examples/programmatic-highlighting/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Charts Embedded SDK Demo: Interactive Filtering</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" type="text/css"
+    href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" />
+  <link href="https://fonts.googleapis.com/css?family=Heebo&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" type="text/css" href="assets/styles.css" />
+</head>
+
+<body>
+  <main>
+    <h2>Change highlights for both charts</h2>
+    <div class="body-content">
+      <div class="chart" id="chart1"></div>
+      <div class="chart" id="chart2"></div>
+    </div>
+    <div class="options-container">
+      <label for="options">Choose a predefined filter document to highlight the chart data, or enter your own MQL
+        filter.</label>
+      <div id="options" class="options">
+        <select id="query-select">
+          <option value="simple">Exact match</option>
+          <option value="range">Using comparisons</option>
+          <option value="complex">Using $and</option>
+          <option value="empty">Applying {}</option>
```

Comment:
Would it be more clear if the text here is 'Remove highlights' instead of "Applying {}"?